connector/mqtt/vernemq/broker/src/dojot_disconnect_plugin/src/utils.erl

```diff
@@ -0,0 +1,57 @@
+-module(utils).
+-import(lists,[member/2]).
+
+-on_load(utils_on_load/0).
+
+% time defined in miliseconds (default 30 min)
+-define(MAX_TIMEOUT, element(1, string:to_integer(os:getenv("PLUGIN_DISC_LIFETIME_SESSION", "1800000")))).
+-define(DISCONNECT_ETS_TABLE, "PLUGIN_DISC_ETS").
+
+-export([
+    set_connection_timeout/1,
+    disconnect_client/1,
+    is_dojot_user/1,
+    cancel_connection_timeout/1
+]).
+
+utils_on_load() -> 
+    try
+        error_logger:info_msg("Loading disconnect plugin utils module"),
+        ets:new(?DISCONNECT_ETS_TABLE, [public, named_table])
```

Comment:
If I understood the `on_load`, it is necessary return `ok`. So, I think it should be added a `ok` after `ets:new(...)`.